debezium-connector-mongodb/src/test/java/io/debezium/connector/mongodb/RecordMakersTest.java

```diff
@@ -161,6 +165,40 @@ public void shouldGenerateRecordForDeleteEvent() throws InterruptedException {
         assertThat(tombstone.valueSchema()).isNull();
     }
 
+    @Test
+    @FixFor("DBZ-582")
+    public void shouldGenerateRecordForDeleteEventWithoutTombstone() throws InterruptedException {
+        RecordMakers recordMakersTemp = recordMakers = new RecordMakers(source, topicSelector, produced::add, false);
```

Comment:
I am not sure I understand the reason for this line

Reply:
I think it stems from the fact that `recordMakers` is set up in the `beforeEach()` method for all the other tests, and just here a specifically configured instance is needed. I'll change it so to just use a local instance of `RecordMakers` and not use the field at all in this method.